src/Microsoft.ML.StandardLearners/Standard/SdcaCatalog.cs

```diff
@@ -60,6 +60,18 @@ public static class SdcaBinaryClassificationExtensions
         /// <param name="l1Threshold">The L1 regularization hyperparameter. Higher values will tend to lead to more sparse model.</param>
         /// <param name="maxIterations">The maximum number of passes to perform over the data.</param>
         /// <param name="advancedSettings">A delegate to set more settings.</param>
+        /// <example>
```

Comment:
Is this duplicated with the section below? It looks like its grabbing range 48-49 which does not grab much.

Reply:
This emphasizes the exact line that SDCA gets called in the sample. 
The second one is the full example.

---
In reply to: [227956416](https://github.com/dotnet/machinelearning/pull/1361#discussion_r227956416) [](ancestors = 227956416)